app/konnect/dev-portal/applications/use-cases/portal-product-versions.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Configuring API Product Versions with App Reg v2
+
+---
+
+
+
+With App Reg v2, developers gain the ability to configure and manage multiple authentication strategies across various API products and their versions, allowing you to apply distinct authentication scopes for different API versions.
+
+This guide will introduce you to using App Reg v2's API endpoints to create Dynamic Client Registration (DCR) within the scenario of managing two different APIs: the Air Traffic API and the Maps API, publishing the latest version to the Flights Dev Portal and configuring different DCR strategies for each API. The examples in the guide serve as templates for your actual requests.
```

Comment:
Sharing some context here to help clarify this in hopes it can be threaded into the docs as needed:
- Existing DCR docs/flows are for the v0 version of DCR
  - UI only and do not have a public supported API to configure them
  - limited in their capability because using DCR is a portal-wide setting that disallows API products from using multiple "strategies" like different DCR providers at once, or some DCR and some non-DCR
  - existing v0 will be completely unavailable with the release of v2, and existing v0 DCR configurations will be automatically migrated to their corresponding v2 data model
- With v2
  - this is the first publicly supported API for DCR
  - allows DCR to be used as granular and flexibly as needed across portals (in the future) and API product versions (with this release)
    - e.g. product version A can use DCR strategy with provider X and product version B can use DCR strategy with provider Y and product version C can use strategy Z (non-DCR). This was not possible in v0

Reply:
> "This guide will introduce you to using App Reg v2's API endpoints"
> 
> Rather than frame it as introducing the new endpoints, what do these new endpoints do? Why would a customer want to use these endpoints and how would they use them in a use case? I'd be more explicit here about what they do because I was kind of confused throughout about what exactly we were doing.

"developers gain the ability to configure and manage multiple authentication strategies across various API products and their versions, allowing you to apply distinct authentication scopes for different API versions."

Today it's either hard (in the case of OIDC) or impossible (in the case of DCR) to use different auth strategies for different API Product versions. These Application Registration APIs let you do that.